pkg/asset-manager-utils/contracts/RewardsAssetManager.sol

```diff
@@ -213,13 +213,7 @@ abstract contract RewardsAssetManager is IAssetManager {
         return _config;
     }
 
-    function getPoolBalances(bytes32 pId)
-        public
-        view
-        override
-        withCorrectPool(pId)
-        returns (uint256 poolCash, uint256 poolManaged)
```

Comment:
I'm not sure how Solidity handles the ABI here: while IAssetManager may call these `poolCash` and `poolManaged`, if we interact with this contract with `RewardsAssetManager` ABI then this may override the interface declaration and drop the names. If so, we should keep them.

Reply:
I think the failing tests may be due to this.